packages/inventory/src/containers/ServicePointProfile/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Profile view Page shows broken page when jurisdiction request errors out: error broken page 1`] = `"Errorlocations.map is not a functionGo BackBack Home"`;
+exports[`Profile view Page shows broken page when jurisdiction request errors out: error broken page 1`] = `"Errorinvalid json response body at  reason: Unexpected token o in JSON at position 1Go BackBack Home"`;
```

Comment:
why do we run into this error, is something wrong in  our tests, Ideally the error that is caught here should be something that we control. for instance as a result of a `fetch.mockReject`